### DISCUSSION TITLE
Ability to disable onboard PR:s through global user GitHub configuration?

### DISCUSSION THREAD

**Discussion by voxpelli** [renovatebot/renovate#22391] [Request Help] [answered]
### I'm using the Mend Renovate hosted app on github.com.

- [X] I'm using the Mend Renovate hosted app.

### Was this something which used to work for you, and then stopped?

I never saw this working

### Describe your problem.

I decided to finally swap from having Renovate only activated on some repositories to have it open on all and experienced two things:

1. It actually did create onboarding PR:s for all forks I have
2. Even not counting them, it became a bit noisy

I would like to have Renovate only run when I have pushed a renovate config myself. I have that added as part of eg. my template repositories for creating new node.js modules and such.

This is a similar request as to eg. https://github.com/renovatebot/renovate/discussions/18826 and https://github.com/renovatebot/renovate/discussions/17134 – would be great if it could be configured by myself through something in either my `.github` or my `renovate-config` repositories on GitHub.

Also: Can you check why I got onboarding PR:s to forks like this one? https://github.com/voxpelli/checkout/pull/1

**Comment by rarkins** [comment]
> Also: Can you check why I got onboarding PR:s to forks like this one? [voxpelli/checkout#1](https://github.com/voxpelli/checkout/pull/1)

The short answer for this is that in the backend of Renovate it has `"repository_selection": "selected"` recorded for your user. This means Renovate thinks you hand-selected each repo instead of installing into "All repositories" and therefore you must want them all to be onboarded. If you switched from "Selected repositories" to "All repositories" then it would seem like our sync has not picked it up (but it has picked up the new repos)

**Reply by voxpelli** [reply]
I guess me turning it off and on again can help it getting in sync then?

**Reply by rarkins** [reply]
It should, yes - it will be a different installation if uninstalled then reinstalled and the data refetched

**Reply by voxpelli** [reply]
@rarkins Oh, I was just thinking of changing the setting from all to selected and back, but I can uninstall and reinstall as well of course. Its a 5 year old installation or something anyhow, so might maybe be good to start fresh. What would happen with all current PR:s? I would like to avoid spamming my repos

**Reply by rarkins** [reply]
As far as I'm aware it makes no change and resumes normally despite being different installations.

**Reply by voxpelli** [reply]
@rarkins Uninstalled and reinstalled it now. How does it look from the backend side? Is `"repository_selection": "selected"` still there?

**Reply by rarkins** [reply]
It's now correctly showing "all" instead of "selected"

**Comment by rarkins** [comment, marked as answer]
Regarding your main question, it sounds like what I have proposed in #22212
